From e8fa1ea6ffae9988be01389c691143ba64125b94 Mon Sep 17 00:00:00 2001
From: Christophe Chapuis <chris.chapuis@gmail.com>
Date: Sat, 6 Jun 2020 13:34:58 +0000
Subject: [PATCH] Make PmLogDaemon compatible with systemd

As per https://www.freedesktop.org/wiki/Software/systemd/syslog , legacy
syslog daemons should now avoid using /dev/log, which is already owned
by systemd-journald.
Instead, /run/systemd/journal/syslog might be used to get all the messages.

Signed-off-by: Christophe Chapuis <chris.chapuis@gmail.com>
Upstream-Status: Pending
---
 src/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.c b/src/main.c
index 7d39849..5263c33 100644
--- a/src/main.c
+++ b/src/main.c
@@ -2655,7 +2655,7 @@ static void InitSettings(void)
 	g_useFullTimeStamps = 0;
 	g_timeStampFracSecDigits = 0;
 
-	strncpy(g_pathLog, _PATH_LOG, sizeof(g_pathLog));
+	strncpy(g_pathLog, "/run/systemd/journal/syslog", sizeof(g_pathLog));
 
 	g_numOutputs = 0;
 	g_numContexts = 0;
-- 
2.37.3.windows.1

